pkg/barrier/barrier.go

```diff
@@ -354,34 +358,30 @@ func generateInjector(f *ci.Framework) {
 				taskIP := *taskStatus.AttemptStatus.PodIP
 				injector.WriteString(taskIP)
 			}
-
 			injector.WriteString("\n")
-			injector.WriteString(
-				"echo " + taskRoleName + "_ips=${" + taskRoleName + "_ips}")
+			injector.WriteString("echo " + ipsEnvName + "=${" + ipsEnvName + "}")
 			injector.WriteString("\n")
 		}
 
-		// {TaskRoleName}_addresses=
-		//   {Task[0].PodIP}:${{TaskRoleName}_port},...,
-		//   {Task[TaskRole.TaskNumber-1].PodIP}:${{TaskRoleName}_port}
+		// FB_{UpperCase({TaskRoleName})}_ADDRESSES=
+		//   {Task[0].PodIP}:${FB_{UpperCase({TaskRoleName})}_PORT},...,
+		//   {Task[TaskRole.TaskNumber-1].PodIP}:${FB_{UpperCase({TaskRoleName})}_PORT}
 		injector.WriteString("\n")
 		for _, taskRoleStatus := range f.TaskRoleStatuses() {
-			taskRoleName := taskRoleStatus.Name
-			injector.WriteString("export " + taskRoleName + "_addresses=")
+			addrsEnvName := getTaskRoleEnvName(taskRoleStatus.Name, "ADDRESSES")
+			portEnvName := getTaskRoleEnvName(taskRoleStatus.Name, "PORT")
 
+			injector.WriteString("export " + addrsEnvName + "=")
 			for _, taskStatus := range taskRoleStatus.TaskStatuses {
 				taskIndex := taskStatus.Index
 				if taskIndex > 0 {
 					injector.WriteString(",")
 				}
-				taskAddr := *taskStatus.AttemptStatus.PodIP +
-						":" + "${" + taskRoleName + "_port}"
+				taskAddr := *taskStatus.AttemptStatus.PodIP + ":" + "${" + portEnvName + "}"
 				injector.WriteString(taskAddr)
 			}
-
 			injector.WriteString("\n")
-			injector.WriteString(
-				"echo " + taskRoleName + "_addresses=${" + taskRoleName + "_addresses}")
+			injector.WriteString("echo " + addrsEnvName + "=${" + addrsEnvName + "}")
```

Comment:
Do we need to add echo in script?

Reply:
Yes, they are like log for injector. We will use them for debug and info user.